koku/koku/rbac.py

```diff
@@ -106,6 +106,13 @@ def _update_access_obj(access, res_access, resource_list):
     """Update access object with access data."""
     for res in resource_list:
         access_items = access.get(res, [])
+        # Set OpenShift rbac inheritance cluster->node->project if higher level is restricted
+        if res == "openshift.node":
+            if not access_items and res_access["openshift.cluster"]["read"] != []:
+                access_items = [{"operation": "read", "resources": ["*"]}]
+        if res == "openshift.project":
+            if not access_items and res_access["openshift.node"]["read"] != []:
+                access_items = [{"operation": "read", "resources": ["*"]}]
```

Comment:
This does require the `resource_list` to be in a specific order. Maybe we should explicitly define the [`RESOURCE_TYPES`](https://github.com/project-koku/koku/blob/2fcd994ba04e69596bb8ebb2a81808ca029565be/koku/koku/rbac.py#L25) as an OrderedDict? I know key order is _now_ maintained in Python, but better safe than sorry?

Reply:
Yeh I wondered this too tbh :+1: